util/strutil/strutil.go

```diff
@@ -0,0 +1,37 @@
+package strutil
```

Comment:
I ended up refactoring some code that used this earlier to not use it. Unless anyone thinks it's worth keeping, I'll toss it out.

Reply:
yeah, these are pretty generic & we should avoid introducing new escaping syntaxes of our own :)